core/src/main/java/org/apache/accumulo/core/data/Value.java

```diff
@@ -48,6 +50,30 @@ public Value() {
   }
 
   /**
+   * Creates a value using the UTF-8 encoding of the CharSequence
+   *
+   * @param cs
+   *          may not be null
+   *
+   * @since 1.8.0
+   */
+  public Value(CharSequence cs) {
+    this(cs.toString().getBytes(StandardCharsets.UTF_8));
+  }
+
+  /**
+   * Creates a Value using the bytes of the Text. Makes a copy, does not use the byte array from the Text.
+   *
+   * @param text
+   *          may not be null
+   *
+   * @since 1.8.0
+   */
+  public Value(Text text) {
+    this(text.getBytes(), 0, text.getLength());
```

Comment:
I was not sure about copying this.  Could implement it such that if the byte array from the text is the correct length, then a copy could be avoided.   I decided against for two reasons.

 * Sometimes copying and sometimes referencing seems confusing
 * The Key constructors that take Text always copy.